HIP/hip-631.md

```diff
@@ -437,7 +492,7 @@ To support these updates the SDK will need to update the available methods on so
     	.addVirtualAddress('0x...') // String
     	.disableVirtualAddress('0x...') // String
     	.removeVirtualAddress('0x...') // String
-    	.setDefaultVirtualAddress(true); // boolean
+    	.setDefaultVirtualAddress('0x...'); // String
```

Comment:
Thanks for this specific change -- I had been asking about it since the very first review.  :-)